samples/Templates/Scenarios/FlightItinerary.template.json

```diff
@@ -108,21 +118,32 @@
 					"type": "Column",
 					"width": 1,
 					"items": [
+						{
+							"type": "TextBlock",
+							"text": "${ArrivesAt}",
```

Comment:
Setting this text to be a template may not be required, consider using a regular string as in "Your Flight Itinerary"